gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -461,10 +467,30 @@ def _monitor_load_jobs(self):
           if load_job.state != 'DONE':
             self._handle_failed_load_job(suffix, load_job)
           else:
+            self._delete_empty_table(suffix, load_job)
```

Comment:
So if a job fails, we keep potentially empty table? Is it because load_job will not have output_rows?

Reply:
If a job fails more than `_BQ_NUM_RETRIES` times then this class raise an exception. At that point it's up to the caller what to do with the newly created not empty tables.

In our case, we call `rollback_newly_created_tables` in `vcf_to_bq` module.